base/util.jl

```diff
@@ -292,7 +292,8 @@ Displays the `message` then waits for user input. Input is terminated when a new
 is encountered or EOF (^D) character is entered on a blank line. If a `default` is provided
 then the user can enter just a newline character to select the `default`.
 
-See also `Base.getpass` and `Base.winprompt` for secure entry of passwords.
+Do not use this function for entering passwords. See `Base.getpass` and `Base.winprompt` for
```

Comment:
Do you maybe want to highlight this more by making it some sort of `!!! warn`-box or `note` or something, similar to the compat annotations?

Reply:
That seems reasonable!

Reply:
Looks like we'll also need to add `Base.getpass` and `Base.winprompt` to the manual otherwise these will be dead links

Reply:
> Looks like we'll also need to add `Base.getpass` and `Base.winprompt` to the manual otherwise these will be dead links

This is done.

Reply:
> Do you maybe want to highlight this more by making it some sort of `!!! warn`-box or `note` or something, similar to the compat annotations?

@dkarrasch Do you know what the Markdown syntax is for this?

Reply:
I guess it's
```julia
!!! warning
    Text
```
, or `!!! tip`, see here: https://juliadocs.github.io/Documenter.jl/stable/showcase/#Admonitions